Week 2 Assignment 2/PizzaShop_W2A2_1.java

```diff
@@ -0,0 +1,106 @@
+import java.util.Scanner;
+
+public class PizzaShop_W2A2_1 {
```

Comment:
Program is great but you need to put the main function in a different class than the Pizza class